siriuspy/siriuspy/devices/orbit_interlock.py

```diff
@@ -18,7 +145,7 @@ class BPMInterlock(_Device):
         # General interlock clear:
         'IntlkClr-Sel',
         # Minimum sum threshold enable:
-        # Habilita interlock de óbita apenas quando threshold da soma
+        # Habilita interlock de órbita apenas quando threshold da soma
         # ultrapassar o valor em "IntlkLmtMinSum-SP"
```

Comment:
portuguese?

Reply:
I kept this documentation temporarily, as it was written during the first few tests, just to keep the information as original as possible.